src/style.css

```diff
@@ -1 +1,79 @@
 /* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  color: #000;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
+}
+
+.nav {
+  position: relative;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav__link {
+  cursor: pointer;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  padding: 23px 0;
+  margin-left: 20px;
+  height: 15px;
+  width: 37px;
+}
+
+.nav__link:hover {
+  text-decoration: underline;
+  text-decoration-color: #00acdc;
+  text-decoration-thickness: 2px;
+  color: #00acdc;
```

Comment:
you typed the #00acdc 3 times 
but what if we decide to use another color?

we would need to find all the places we use it and change
if you use CSS variable you can change the color in 1 place